app/assets/javascripts/discovery-two-pane.js

```diff
@@ -0,0 +1,7 @@
+$(document).on('click', ".table-two-pane td[class$='display-two-pane'] a", function(e) {
```

Comment:
This can be removed after https://github.com/theforeman/foreman/pull/2048 is merged.
It's possible to merge this PR and create a refactoring issue to fix it instead of waiting for the change in core.